packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1319,10 +1319,19 @@ export interface TaskDto {
     problemMatcher?: any;
     group?: string;
     detail?: string;
+    presentation?: TaskPresentationOptionsDTO;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     [key: string]: any;
 }
 
+export interface TaskPresentationOptionsDTO {
+    reveal?: number;
```

Comment:
I know that a `number` takes less space than a string, but is this really useful for this DTO? I genuinely don't know, does VS Code expect this type? If not it would spare us the conversion map.

Reply:
Yeap, vscode expect it from the plugins as: 
![image](https://user-images.githubusercontent.com/76971376/112884689-cdaf1200-909d-11eb-841a-21184e7eb8e9.png)